test/smoke/src/sql/areas/queryEditor/queryEditor.test.ts

```diff
@@ -38,6 +38,13 @@ export function setupWeb(opts: minimist.ParsedArgs) {
 function setupCommonTests(opts: minimist.ParsedArgs): void {
 	beforeSuite(opts);
 	afterSuite(opts);
+
+	beforeEach(async function (): Promise<void> {
+		const app = this.app as Application;
+		// Clearing any toast notifications that popup when ADS is launched like Telemetry opt-out.
+		await app.workbench.notificationToast.closeNotificationToasts();
```

Comment:
We should launch the process with `--disable-telemetry` instead, otherwise this could be closing other notification toasts that aren't expected to be there (and should fail the tests if they show up)

Reply:
@Charles-Gagnon . it seems that this PR broke the flag #18498.

Reply:
We are still seeing the telemetry popup when launched with --disable-telemetry
https://github.com/microsoft/azuredatastudio/blob/2bc6a0cd012de69504061a876070cfa43533fedc/test/automation/src/code.ts#L137

Reply:
Well that's not good, we should fix that then.